src/libraries/System.Net.Http/src/System/Net/Http/HttpClientHandler.AnyMobile.cs

```diff
@@ -73,6 +65,23 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
+        [CLSCompliant(false)]
+        public Meter Meter
+        {
+            get => _meter;
+            set
+            {
+                ArgumentNullException.ThrowIfNull(value);
+                if (value.Name != "System.Net.Http")
+                {
+                    throw new ArgumentException("Meter name must be 'System.Net.Http'.");
+                }
```

Comment:
https://github.com/dotnet/runtime/issues/86961#issuecomment-1598115821

> Think of the meter name like a namespace. It's a way to group counters together. Also, when counter values are exported to Prometheus (a cloud native metrics database), the meter name is included as a tag. The meter name is externally visible in the exported data. The OTel spec might provide insight of how things relate to each other: [open-telemetry/opentelemetry-specification@`main`/specification/metrics/api.md](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md?rgh-link-date=2023-06-20T04%3A56%3A44Z)
> 
> We want to restrict SocketsHttpHandler and friends to only allow "System.Net.Http" as the meter name to prevent people from changing the names of built-in counter values when they're exported and queried. It simplifies the eco-system if we lock it down.

Reply:
I hear the argument, I just don't agree with it. We're providing an extensibility mechanism and then artificially limiting it unnecessarily.